Mathlib/Data/Set/Intervals/Group.lean

```diff
@@ -5,12 +5,14 @@ Authors: Johannes Hölzl, Mario Carneiro, Patrick Massot, Yury Kudryashov, Rémy
 Ported by: Winston Yin
 
 ! This file was ported from Lean 3 source module data.set.intervals.group
-! leanprover-community/mathlib commit 18a5306c091183ac90884daa9373fa3b178e8607
+! leanprover-community/mathlib commit 740acc0e6f9adf4423f92a485d0456fc271482da
 ! Please do not edit these lines, except to modify the commit id
 ! if you have ported upstream changes.
 -/
 import Mathlib.Data.Set.Intervals.Basic
 import Mathlib.Algebra.Order.Group.Abs
+import Mathlib.Data.Set.Pairwise
+import Mathlib.Algebra.GroupPower.Lemmas
```

Comment:
This import order isn't the same as the mathlib3 order:
```suggestion
import Mathlib.Data.Set.Intervals.Basic
import Mathlib.Data.Set.Pairwise
import Mathlib.Algebra.Order.Group.Abs
import Mathlib.Algebra.GroupPower.Lemmas
```

(from https://github.com/leanprover-community/mathlib3port/blob/6e37b6d7464082351091ed0a9fb032205c21aae2/Mathbin/Data/Set/Intervals/Group.lean#LL11-L14C41)